pkg/modules/user/impluser/handler.go

```diff
@@ -92,16 +92,15 @@ func (h *handler) CreateInvite(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	_, err = h.module.CreateBulkInvite(ctx, claims.OrgID, claims.UserID, &types.PostableBulkInviteRequest{
+	invites, err := h.module.CreateBulkInvite(ctx, claims.OrgID, claims.UserID, &types.PostableBulkInviteRequest{
 		Invites: []types.PostableInvite{req},
 	})
 	if err != nil {
 		render.Error(rw, err)
 		return
 	}
 
-	render.Success(rw, http.StatusCreated, nil)
-	return
+	render.Success(rw, http.StatusCreated, &types.GettableCreateInviteResponse{InviteToken: invites[0].Token})
```

Comment:
```suggestion
	render.Success(rw, http.StatusCreated, &types.GettableCreateInviteResponse{InviteToken: invites[0].Token})
```

Send the entire invite here please. `ID` is mandatory.